include/odp/api/spec/packet_io_types.h

```diff
@@ -917,6 +917,11 @@ typedef struct odp_pktio_capability_t {
 	/** Statistics counters capabilities */
 	odp_pktio_stats_capability_t stats;
 
+	/** This defines whether the pktio supports moving received packets to
+	 * the pool indicated by Class of Service. @see
+	 * odp_cls_cos_param_t::pool */
+	odp_bool_t cls_cos_pool;
+
```

Comment:
This would need to be part of odp_cls_capability_t.

It's another topic to decide if per pktio interface cls capa is needed. Maybe it is and then we could add odp_cls_capability_t into odp_pktio_capability_t.